pkg/kubelet/metrics/metrics.go

```diff
@@ -79,6 +79,19 @@ const (
 	// Metrics keys for RuntimeClass
 	RunPodSandboxDurationKey = "run_podsandbox_duration_seconds"
 	RunPodSandboxErrorsKey   = "run_podsandbox_errors_total"
+
+	// Metrics to keep track of objects under management
+	ManagedPodsKey                  = "managed_pods"
+	ManagedContainersKey            = "managed_containers"
+	StartedPodsTotalKey             = "started_pods_total"
+	StartedPodsErrorsTotalKey       = "started_pods_errors_total"
+	StartedContainersTotalKey       = "started_containers_total"
+	StartedContainersErrorsTotalKey = "started_containers_errors_total"
+
+	// Labels used in metric fields
+	ContainerLabel          = "container"
+	InitContainerLabel      = "initContainer"
```

Comment:
Nit: This should be `init_container` instead, same for `ephemeralContainer` below -> `ephemeral_container`.

Reply:
Ok, updated these to `init_container` and `ephemeral_container`.

I think I used the wrong terminology in the comment (I'll go back and fix this), so just to make sure I didn't mislead: these are the values used in the label named `container_type`. I arbitrarily chose the names used in the API (`container`, `initContainer`, `ephemeralContainer`)  because I didn't know the convention for label values.

So previously it was:

```
# HELP kubelet_started_containers_total [ALPHA] Cumulative number of containers started
# TYPE kubelet_started_containers_total counter
kubelet_started_containers_total{container_type="container"} 5
kubelet_started_containers_total{container_type="initContainer"} 1
```

and now it's:
```
# HELP kubelet_started_containers_total [ALPHA] Cumulative number of containers started
# TYPE kubelet_started_containers_total counter
kubelet_started_containers_total{container_type="container"} 5
kubelet_started_containers_total{container_type="init_container"} 1
```

Reply:
Looks good to me, thank you!